src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.csproj

```diff
@@ -98,6 +98,8 @@
     <Compile Include="$(CommonTestPath)\System\Collections\IEnumerable.Generic.Serialization.Tests.cs">
       <Link>Common\System\Collections\IEnumerable.Generic.Serialization.Tests.cs</Link>
     </Compile>
+    <Compile Include="ConcurrentBagTests.netstandard1.7.cs" />
```

Comment:
These are supposed to be named with netstandard1.7? It's somewhat confusing that in the ref file the members are conditionally included on `#if netcoreapp11` but we include the tests for any netstandard1.7.